apps/browser-extension-wallet/src/utils/validators/address-book.ts

```diff
@@ -75,6 +92,45 @@ export const validateWalletHandle = async ({ value, handleResolver }: validateWa
   return '';
 };
 
+type hasHandleOwnerChangedArgs = {
+  value: string;
+  address: Cardano.PaymentAddress;
+  handleResolver: HandleProvider;
+};
+
+export const hasHandleOwnerChanged = async ({
+  value,
+  address,
+  handleResolver
+}: hasHandleOwnerChangedArgs): Promise<void> => {
```

Comment:
```suggestion
}: hasHandleOwnerChangedArgs): Promise<boolean> => {
```
Semantically I'd expect this response type based on the naming. Alternatively: `ensureHandleOwnerHasntChanged`

Reply:
a814ce1d9877e036403c6119d5d7f6c050ff2862 should address this

Reply:
I can see the return type was changed but the implementation remains the same, despite there being [a test](https://github.com/input-output-hk/lace/commit/a814ce1d9877e036403c6119d5d7f6c050ff2862#diff-dc73c6511012a53f13d8c3682f5b60e0674eac58c8d3c55814658224b77ae06cR290) that describes a different outcome :thinking:

Reply:
yea, fixed here [0328590](https://github.com/input-output-hk/lace/pull/262/commits/0328590351b6757109c1b7f0707169349e290109)

Reply:
Is there a test to check it returns `true`?

Reply:
That's fixed here: a463d6844a1e9ae2f2c485be84e1908808f249bd